### ISSUE TITLE
[DOCS ISSUE] Page: Page Templates

### ISSUE THREAD

**Issue by pwohlschl** [apostrophecms/apostrophe-documentation#302]
The documentation invites to edit "app.js", when actually the file to update is the relevant "index.js".

**Comment by abea** [comment]
Are you referring to "### Example: Adding a Page Template to `app.js`"? That is true, but not the best practice. I'll update that. Please let me know if you're talking about another specific reference.

**Comment by abea** [comment]
Also "2. Register the new page template in `app.js`". The same applies but, again, it's not the best practice anymore.

**Comment by pwohlschl** [comment]
Correct, I first came across this issue whilst going through the tutorials online. I spotted it on a few occasions on "Page Templates": 
- "2. Register the new page template in app.js"
- "Now that you have a default.html file, you need to register it in app.js to make it available."
- "#Example: Adding a Page Template to app.js"

I think this change in best practice might have impacted other parts of the documentation, e.g. in the "Building Navigation" pages, I also read "first, in app.js, let's configure apostrophe-pages to retrieve two levels of children when fetching ancestors of the current page".
